docs/unified/novacustom/initial-deployment.md

```diff
@@ -0,0 +1,201 @@
+# Initial Deployment
+
+## Intro
+
+This document is a guide for the initial installation of Dasharo on a supported
+device. It assumes some knowledge about external flashing and is primarily aimed
+at technicians performing initial installation and not for end users of the
+devices.
+
+## Installing Dasharo
+
+### Initial Installation
+
+=== "12th Gen (Alder Lake)"
+
+    During the initial installation of Dasharo, you should deploy the supported
+    Intel ME version (and configuration) on the device. Since vendor firmware
+    has enabled Intel Boot Guard and BIOS Guard, it is not possible to do this
+    from within the operating system and external flashing of the whole flash
+    chip using a programmer like the CH341a or
+    [3mdeb RTE](https://docs.dasharo.com/transparent-validation/rte/v1.1.0/flashing-guide/#ns5x7x-12th-generation)
+    is required.
+
+    > Publicly released binaries do not contain ME binary. If you need an Intel ME
+    > update for your device, contact us via already established commercial support
+    > channel.
+
+    ### Preparation
+
+    Install flashrom:
+
+    ```bash
+    apt update
+    apt upgrade
+    sudo apt -y install flashrom
+    ```
+
+    ### BIOS installation
+
+    Steps for installing Dasharo BIOS:
+
+    1. Remove the bottom cover of the laptop.
+
+        === "NS5x / NS7x"
+            ![ns5x chips](../../images/ns50mu_board_chips.jpg)
+
+        === "NV4x"
+            ![nv4x chips](../../images/nv4x_board_chips.jpg)
+
+    1. Disconnect the primary battery. (1)
```

Comment:
These steps are duplicated here and in the linked RTE document

Reply:
Question is, should they be here or in the RTE documentation?